app/services/websites/create.rb

```diff
@@ -14,7 +14,15 @@ def initialize(params, user)
     end
 
     def call
-      if params[:website][:url].to_s.include? '.'
+      already_exist = Website.find_by(url: params[:website][:url])
+      if already_exist && params[:website][:collection_id]
+        website = CollectionWebsite.where('collection_id = ?', params[:website][:collection_id])
+                                   .where('website_id = ?', already_exist.id)
+                                   .first
+        if website.nil?
+          CollectionWebsite.create!(collection_id: params[:website][:collection_id], website_id: already_exist.id)
+        end
```

Comment:
Ok, go ahead with this for now - we can refactor later, but make sure there's a test case that covers the issue that is solved by this change, ie. the test should fail if you replaced this code with the old version, but putting this code should make that test pass.